source/triggers/database-triggers.txt

```diff
@@ -591,22 +594,28 @@ multiple executions of a Trigger Function occur in event order.
 
 Disable Collection-Level Preimages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-          
-Document preimages use extra information stored in the oplog.
-The extra data may degrade trigger performance.
 
-Once you've enabled document preimages for any trigger on a
-given collection, that collection will include preimage data in
-the oplog and other triggers on the collection can use preimages
-with no additonal overhead.
+Document preimages require your cluster to record additional data about
+each operation on a collection. The additional storage space and compute
+overhead may degrade trigger performance depending on your cluster
+configuration.
+
+Once enabled, your cluster continues to store preimages for a collection
+unless you explicitly disable them at the collection level. This means
+that your cluster will record preimages even if you delete all triggers
+on the collection.
```

Comment:
i don't understanding this paragraph. could you clarify?

Reply:
Updated